src/app/sessions/containers/sessions-propositions-search/sessions-propositions-search.component.ts

```diff
@@ -22,6 +26,7 @@ import { combineLatest } from 'rxjs/observable/combineLatest';
 })
 export class SessionsPropositionsSearchComponent implements OnInit {
```

Comment:
SessionsPropositionsSearchComponent is our smart container here and has all the data that CreateSessionForm requires. Why do we then need to use SessionsCreateComponent (another smart container) inside  SessionEditOrCreateDialogComponent?

now we have: smart -> dumb -> smart -> dumb
why not simply: smart(SessionsPropositionsSearchComponent) -> dumb(SessionsCreateFormComponent) ?